docs/FAQ.md

```diff
@@ -85,6 +85,10 @@ cd $(brew --repository)
 hub pull someone_else
 ```
 
+## Why should I install Homebrew in the default location?
+
+Homebrew's pre-built binary packages, called [bottles](Bottles.md), of many packages can only be used if you install in the default installation prefix, otherwise they have to be built from source. Building from source takes a long time, is prone to fail, and is not supported. Do yourself a favour and install to the default prefix so that you can use our pre-built binary packages. The default prefix is `/usr/local` for macOS on Intel, `opt/homebrew` for macOS on ARM, and `/home/linuxbrew/.linuxbrew` for Linux. *Pick another prefix at your peril!*
```

Comment:
```suggestion
Homebrew's pre-built binary packages, called [bottles](Bottles.md), of many packages can only be used if you install in the default installation prefix, otherwise they have to be built from source. Building from source takes a long time, is prone to fail, and is not supported. Do yourself a favour and install to the default prefix so that you can use our pre-built binary packages. The default prefix is `/usr/local` for macOS on Intel, `/opt/homebrew` for macOS on ARM, and `/home/linuxbrew/.linuxbrew` for Linux. *Pick another prefix at your peril!*
```

Reply:
Looks good, might be worth checking or linking `Installation.md` which covers some of this.